deployment/gunicorn/gunicorn_conf.py

```diff
@@ -23,8 +23,13 @@ def post_fork(server, worker):
     resolve('/')
 
 
-def on_starting(server, path=None):
-    """Wipe the metrics from previous processes"""
+def on_starting(server):
+    """Function for testing since actual function can not have additional args"""
+    _on_starting(server)
+
+
+def _on_starting(server, path=None):
```

Comment:
`path` will always be `None` in production. Why do we need this parameter? Or maybe the question is why test with paths other than `None` if that will never happen in production?

Reply:
it just made testing easier but I could also have patched `os.environ`